sysvinit/openbsd/cloudconfig.tmpl

```diff
@@ -0,0 +1,39 @@
+## template:jinja
```

Comment:
Why is this a template? (same comment on the others)

Reply:
Because it is using this in each file:

`daemon_execdir="{{prefix}}/bin"`